internal/frr/templates/filters.tmpl

```diff
@@ -75,9 +75,22 @@ ip prefix-list {{$plistName}} seq {{counter $plistName}} deny any
 {{- end }}
 {{ if not .neighbor.Incoming.PrefixesV6 }}
 ipv6 prefix-list {{$plistName}} seq {{counter $plistName}} deny any
+{{- end }}
+
+{{- if .neighbor.AlwaysBlock}}
+
+{{ $plistName:=deniedIncomingList $.neighbor}}
+{{- range $a := .neighbor.AlwaysBlock }}
+{{frrIPFamily $a.IPFamily}} prefix-list {{$plistName}} seq {{counter $plistName}} permit {{$a.Prefix}}{{$a.Matcher}}
```

Comment:
nit/question: why is this permit and not deny?

Reply:
permit, in the context of a prefix list, means the prefix is added to the prefix list. If we were adding deny, then the prefix would not be added and thus ignored by the rule